packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -233,6 +250,7 @@ export class StripeHelper {
       this.stripe,
       firestore_prefix
     );
+    this.paymentConfigManager = Container.get(PaymentConfigManager);
```

Comment:
I didn't because it didn't seem risky since we already use Firestore in production.

It's essentially inert.  But maybe there are risks I haven't considered.

Reply:
Why would this be risky to use the feature flag?

We are adding a bunch of logic and new Firestore collections and operations that affect a route that is hit every time someone goes to the payments server. It seems risky to omit the flag IMO, since the page simply won't work if there's an unhandled error.

Reply:
Ok, how about this: we wrap this in a feature flag but set the flag to true on stage?

But what are the conditions for this to be considered safe?  Is having it immediately on prod your concern?

Reply:
Yeah I just worry that this will go to prod in 1 week and we won't have finished the epic and therefore done comprehensive QA (honestly haven't thought too much what this will look like for this epic), and we hit some unexpected error.

Reply:
If having it on stage longer than a week feels safer, then sure let's do that.